Doc/install/INSTALL.rst

```diff
@@ -0,0 +1,95 @@
+======================
+Biopython Installation
+======================
+
+This document describes installing Biopython on your computer. To make things as simple as possible, 
+it basically assumes you have nothing related to Python or Biopython on your computer and want to 
+end up with a working installation of Biopython. Biopython supports Mac, UNIX/Linux and Windows machines, but the 
+steps to install biopython vary across the platforms.
+
+
+C Compiler
+==========
+
+1) Unix/Linux
+-------------
+
+GCC is recommende as the C compiler, which is usually available as part of the standard set of packages on any Unix or Linux system.
+
+2) Mac OS X
+------------
+
+Install Apple’s XCode suite from the App Store, and then from the Xcode options also install the optional command line utilities.
+
+3) Windows
+-----------
+
+We recommend you install Biopython and its dependencies using the provided pre-compiled Windows Installers. i.e. You don’t need a C compiler. 
+See subsequent section for more details.
+
+Installing Python
+=================
+
+Python is a interpreted, interactive object-oriented programming language and the home for all things python is `https://www.python.org. <https://www.python.org/>`_
+
+Biopython is designed to work with Python 2.7, Python 3.4, Python 3.5 and 3.6. As of Biopython 1.68, support for Python 3.3 has been removed.
+
+1) Installing Python in UNIX/Linux
+----------------------------------
+
+Python 2.7 and Python 3.5 usually comes pre-installed with most modern Linux systems like Ubuntu, Fedora. If you would like to check 
+the version present in your system, you can type the command::
+
+  $ python --version
+  
+This command will show the version of python2 present in your system. To check the exact version of python3 interpreter, type::
+
+  $ python3 --version
+    
+This command will show the version of python3 present in your system. If you aren't able to find the Python software available 
+in your Linux system, you can download Python from the site here- `https://www.python.org/downloads/source/ <https://www.python.org/downloads/source/>`_ and select the source gzipped tarball for the version you would like to install.
+For example, If you want to install Python 3.6 in your system, Look for the latest stable Python 3.6 release and Click on the "Download Gzipped source tarball".
```

Comment:
I don't mind linking the python.org for instructions, but wouldn't it be an issue if we tell users to go to some link everytime for some software or for a package( like we can tell users to go to NumPy docs link for instead of explaining the installation ourselves).